chainer/__init__.py

```diff
@@ -20,6 +21,7 @@
 from chainer import serializers  # NOQA
 from chainer import training  # NOQA
 from chainer import variable  # NOQA
+from chainer import warnings  # NOQA
```

Comment:
Doesn't this overwrite `import warnings` above?

Reply:
Oops